package.json

```diff
@@ -12,9 +12,9 @@
     "build": "tsc --build packages/*",
     "clean": "yarn run build --clean",
     "prettier": "prettier --write ./packages/**/* ./docs/**/* *.md",
-    "integration:local": "./env.sh yarn run _integration:local",
-    "_integration:local": "yarn run _integration:prep && jest --config=jest.config.integration.js",
-    "_integration:prep": "ts-node scripts/integrationPrep.ts"
+    "integration:postgres": "./env.sh yarn run _integration:postgres",
```

Comment:
I feel like we should move this into the `adapter-postgres` package.

Reply:
If you have this as a `integration` script inadapter-postgres, you can use `yarn lerna run integration`, and it'll run that script in all of the packages that have it.